Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift

```diff
@@ -77,34 +77,34 @@ final class PledgeCreditCardView: UIView {
     super.bindStyles()
 
     _ = self
-      |> viewStyle
+      |> self.viewStyle
```

Comment:
Hmm why did it add `self` here? These styles should live outside the class, private to the file. 🤔